internal/app/admin/http/handler.go

```diff
@@ -0,0 +1,136 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	"github.com/envoyproxy/xds-relay/internal/app/cache"
+
+	"github.com/envoyproxy/xds-relay/internal/app/orchestrator"
+
+	bootstrapv1 "github.com/envoyproxy/xds-relay/pkg/api/bootstrap/v1"
+)
+
+type Handler struct {
+	prefix      string
+	description string
+	handler     http.HandlerFunc
+}
+
+func getHandlers(bootstrap *bootstrapv1.Bootstrap, orchestrator *orchestrator.Orchestrator) []Handler {
+	handlers := []Handler{
+		{
+			"/",
+			"admin home page",
+			defaultHandler(nil),
```

Comment:
can we omit defining this here if we define it [later](https://github.com/envoyproxy/xds-relay/pull/74/files#diff-29e3fee34912e53bdb13be1567ad0307R43)?

Reply:
Switched to an empty HandlerFunc. I include it in the list so it shows up in the output for the homepage.